stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -40,6 +44,10 @@ internal class PaymentSheetViewModel internal constructor(
         mutableTransition.postValue(target)
     }
 
+    fun setSelection(selection: Selection?) {
```

Comment:
Maybe `updateSelection()`? I also confused this with https://developer.android.com/reference/android/text/Selection#setSelection(android.text.Spannable,%20int,%20int) at first